client/src/components/RnR/ReviewsButtons.jsx

```diff
@@ -50,4 +54,4 @@ class ReviewsButtons extends React.Component {
   }
 }
 
-export default ReviewsButtons;
\ No newline at end of file
+export default HOC(ReviewsButtons, 'RnR');
```

Comment:
Is the second argument to the HOC supposed to correspond exactly with the Business Docs?   Or can you just put the name of your widget as you named it in there?